ui/src/styles.scss

```diff
@@ -44,6 +44,36 @@ body {
   color: #3367d6;
 }
 
```

Comment:
There are other common css that can be moved to styles.scss too, will this PR address only subset of those or is it supposed to have all the common css movement to styles.scss ?

Reply:
Consider this as a subset of common css. We will move in the comming  backlog PRs.